subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/internal/report/listener/TestArtifactsUploader.kt

```diff
@@ -0,0 +1,13 @@
+package com.avito.instrumentation.internal.report.listener
+
+import com.avito.android.Result
+import com.avito.report.model.Entry
+import okhttp3.HttpUrl
+import java.io.File
+
+internal interface TestArtifactsUploader {
+
+    suspend fun uploadLogcat(logcat: String): Result<HttpUrl>
```

Comment:
Generic uploader API knows about a specific use case.

Let's express this as string\log uploading.

Reply:
it's not generic, it's TestArtifacts uploader )
but i agree, it could be more generic

Reply:
fixed